hdInsight-apache-spark-1-4-1/azuredeploy.json

```diff
@@ -0,0 +1,226 @@
+            ],
+            "tags": { },
+            "properties": {
+                "clusterVersion": "3.2",
+                "osType": "Linux",
+                "clusterDefinition": {
+                    "kind": "[variables('clusterType')]",
+
+                    "configurations": {
+                        "gateway": {
+                            "restAuthCredential.isEnabled": true,
+                            "restAuthCredential.username": "[parameters('LoginUsername')]",
+                            "restAuthCredential.password": "[parameters('LoginPassword')]"
+                        },
+                        "hive-site": {
+                            "hive.metastore.client.connect.retry.delay": "5",
+                            "hive.execution.engine": "mr",
+                            "hive.security.authorization.manager": "org.apache.hadoop.hive.ql.security.authorization.DefaultHiveAuthorizationProvider"
+                        }
+                    }
+                },
+                "storageProfile": {
+                    "storageaccounts": [
+                        {
+                            "name": "[concat(variables('clusterStorageAccountName'),'.blob.core.windows.net')]",
+                            "isDefault": true,
+                            "container": "[parameters('clusterName')]",
+                            "key": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('clusterStorageAccountName')), variables('defaultApiVersion')).key1]"
+                        }
+                    ]
+                },
+                "computeProfile": {
+                    "roles": [
+                        {
+                            "name": "headnode",
+                            "targetInstanceCount": "2",
+                            "hardwareProfile": {
+                                "vmSize": "[parameters('clusterNodeSize')]"
+                            },
+                            "osProfile": {
+                                "linuxOperatingSystemProfile": {
+                                    "username": "[parameters('LoginUsername')]",
+                                    "password": "[parameters('LoginPassword')]"
+                                }
+                            },
+                            "virtualNetworkProfile": {
+                                "id": "[resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName'))]",
+                                "subnet": "[concat(resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName')), '/subnets/', variables('clusterVNetSubnetName'))]"
+                            },
+                            "scriptActions": [
+                                {
+                                    "name": "Apache Spark 1.4.1",
+                                    "uri": "https://jaidistro.blob.core.windows.net/main/spark141-installer-v04.sh",
```

Comment:
Can this script live in this Github repo so that in case the above storage account is removed the template does not fail?